### PR TITLE
Add Peakzilla

### PR DESCRIPTION
**Pull request by npinter**
Peakzilla identifies sites of enrichment and transcription factor binding sites from transcription factor ChIP-seq and ChIP-exo experiments at hight accuracy and resolution.

### CODE REVIEW

**Comment by bgruening** [comment]
Hi @npinter!

```
Failed linting
Applying linter xml_order... WARNING
.. WARNING: Best practice violation [macros] elements should come before [stdio]
Applying linter tests... WARNING
.. WARNING: Test should specify 'expect_num_outputs' if outputs have filters
.. CHECK: 1 test(s) found.
Applying linter output... WARNING
.. WARNING: Unknown element found in outputs [filter]

Applying linter help... WARNING
.. WARNING: Invalid reStructuredText found in help - [<string>:9: (SEVERE/4) Unexpected section title.

INPUT FORMAT
------------
].

.. ERROR: Invalid XML found in file: tmp8f0d_bi7. Errors [/tmp/tmp8f0d_bi7:60:0:ERROR:SCHEMASV:SCHEMAV_ELEMENT_CONTENT: Element 'filter': This element is not expected.
```

All warnings and errors need to be fixed before we can merge.

File: tools/peakzilla/peakzilla.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+            <param name="chip_bed" value="chip.bed" />
+            <param name="input_bed" value="input.bed" />
+            <param name="model_peaks" value="200" />
+            <param name="fragment_size" value="50" />
+            <param name="enrichment_cutoff" value="2" />
+            <param name="score_cutoff" value="1" />
+            <param name="log" value="true" />
+            <param name="negative" value="true" />
+            <output name="results" file="results.tsv">
+                <asserts>
```
Comment (bgruening) [inline tools/peakzilla/peakzilla.xml:83]:
```suggestion
                <assert_contents>
```

File: tools/peakzilla/peakzilla.xml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                    <has_text>Peak_1</has_text>
+                </asserts>
```
Comment (bgruening) [inline tools/peakzilla/peakzilla.xml:85]:
```suggestion
                </assert_contents>
```

File: tools/peakzilla/peakzilla.xml [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+            </output>
+            <output name="log" file="log.txt">
+                <asserts>
```
Comment (bgruening) [inline tools/peakzilla/peakzilla.xml:88]:
```suggestion
                <assert_contents>
```

File: tools/peakzilla/peakzilla.xml [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+                    <has_text>Peakzilla</has_text>
+                </asserts>
```
Comment (bgruening) [inline tools/peakzilla/peakzilla.xml:90]:
```suggestion
                </assert_contents>
```

File: tools/peakzilla/peakzilla.xml [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+            </output>
+            <output name="negative_peaks" file="negative_peaks.tsv">
+                <asserts>
```
Comment (bgruening) [inline tools/peakzilla/peakzilla.xml:93]:
```suggestion
                <assert_contents>
```

File: tools/peakzilla/peakzilla.xml [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+                    <has_text>Peak_1</has_text>
+                </asserts>
```
Comment (bgruening) [inline tools/peakzilla/peakzilla.xml:95]:
```suggestion
                </assert_contents>
```

**Comment by npinter** [comment]
thanks :) @bgruening

**Review by bgruening — APPROVED** [review]

**Pull request merged**